homework-4/style.css

```diff
@@ -0,0 +1,175 @@
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
+a {
+  text-decoration: none;
+}
+button {
+  border: none;
+  background: transparent;
+}
+.container {
+  max-width: 1440px;
+  margin: 0 auto;
+}
+.d-flex{
```

Comment:
bad class name. I suppose it will add only `display: flex;`, but suddenly I get paddings, margins and even color.

Reply:
change